codecov.yaml

```diff
@@ -1,6 +1,6 @@
 codecov:
   require_ci_to_pass: true
-  branch: master
+  branch: development
```

Comment:
I don't know if this will work because this repo doesn't have a development branch. (sbc-auth doesn't either and I copied this from there.)